pkg/caveats/errors.go

```diff
@@ -7,6 +7,21 @@ import (
 	"github.com/rs/zerolog"
 )
 
+// EvaluationErr is an error in evaluation of a caveat expression.
+type EvaluationErr struct {
```

Comment:
Seems similar to the error in `internal/caveats`. Any chances we can move the one internal here and import it from `internal/caveats`?

Reply:
Nope; this was explicitly done (and hence the rename above) because it needs to add additional context to it